src/components/MaterialUI/DataDisplay/PredefinedElements/InformationItem.js

```diff
@@ -26,10 +25,14 @@ const useStyles = makeStyles(theme => ({
 		color: theme.palette.grey.dark,
 		fontFamily: theme.typography.fontFamily,
 	},
+	container: {
+		width: "100%",
+		marginTop: props => theme.spacing(props.mt),
+	},
 }));
 
-const InformationItem = ({ label, children, required, error, showNotAvailable = false }) => {
-	const classes = useStyles({ required, error });
+const InformationItem = ({ label, children, required, error, showNotAvailable = false, mt = 0 }) => {
```

Comment:
Curious, what does mt stand for?  Margin Top?  Maybe the prop name can be marginTop?